### PR TITLE
Fix shape of data in RNN tutorial

### LINKED ISSUE #218: RNN example in tutorial is broken
I had to replace

    h = Variable(np.zeros((50,), dtype=np.float32), volatile=volatile)
    loss = 0

with

    N = x_list.shape[1]  # minibatch size
    h = Variable(np.zeros((N,50), dtype=np.float32), volatile=volatile)
    loss = Variable(np.zeros(N))

### PR DESCRIPTION
**Pull request by unnonouno**
fix #218

### CODE REVIEW

File: docs/source/tutorial/recurrentnet.rst [hunk 1]
```diff
@@ -58,13 +58,15 @@ The forward computation is simply written with a for loop::
...
   def forward(x_list, volatile=False):
-      h = Variable(np.zeros((50,), dtype=np.float32), volatile=volatile)
+      h = Variable(np.zeros((1, 50), dtype=np.float32), volatile=volatile)
       loss = 0
       for cur_word, next_word in zip(x_list, x_list[1:]):
           h, new_loss = forward_one_step(h, cur_word, next_word, volatile=volatile)
           loss += new_loss
       return loss
 
+Note that the first dimention of ``h`` and ``x_list`` is always the mini-batch size.
```
Comment (beam2d) [inline docs/source/tutorial/recurrentnet.rst]: dimention -> dimension

**Comment by beam2d** [comment]
LGTM except one comment.

**Comment by unnonouno** [comment]
fixed!

**Comment by beam2d** [comment]
:+1:
